src/goto-programs/goto_trace.cpp

```diff
@@ -386,14 +386,8 @@ void show_goto_trace(
       break;
 
     case goto_trace_stept::typet::CONSTRAINT:
-      UNREACHABLE;
```

Comment:
This kind of change probably at least requires something in the commit message as to why these are now reasonable states to reach (Someone obviously thought they were unreachable before...)

Reply:
`UNREACHABLE` is still present (line 392)

Reply:
Ah of course as it now falls through